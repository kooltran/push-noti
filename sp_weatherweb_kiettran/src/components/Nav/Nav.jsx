import React from "react";

const Nav = () => {
  return (
    <div className="nav-container">
      <div className="nav-burger">
        <span></span>
        <span></span>
        <span></span>
      </div>
      <div className="nav-location">
        <div className="nav-location-name">myENV</div>
        <div className="nav-location-dropdown">
          <span>Current Location</span>
          <span className="icon-arrow"></span>
        </div>
      </div>
      <div className="nav-notification"></div>
    </div>
  );
};

export default Nav;
