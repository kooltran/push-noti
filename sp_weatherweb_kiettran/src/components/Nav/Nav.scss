.nav {
  &-container {
    display: flex;
    justify-content: space-between;
    padding: 20px;
  }

  &-burger {
    display: inline-block;
    width: 30px;
    height: 35px;
    cursor: pointer;

    span {
      display: block;
      width: 100%;
      height: 3px;
      background-color: $COLOR_WHITE;
      margin: 6px 0;
      transition: 0.4s;
    }
  }

  &-location {
    text-align: center;

    &-dropdown {
      display: flex;
      align-items: center;
    }

    .icon-arrow {
      display: inline-block;
      width: 15px;
      height: 15px;
      background-size: contain;
      background-image: url($ICON_ARROW_DOWN);
      margin-left: 5px;
    }
  }

  &-notification {
    width: 30px;
    height: 30px;
    background-size: contain;
    background-image: url($ICON_BELL);
    cursor: pointer;
  }
}
