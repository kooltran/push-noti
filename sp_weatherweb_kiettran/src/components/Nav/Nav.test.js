import React from "react";
import renderer from "react-test-renderer";
import Nav from "./Nav";

it("should render Nav correctly", () => {
  const tree = renderer.create(<Nav />).toJSON();
  expect(tree).toMatchSnapshot();
});
