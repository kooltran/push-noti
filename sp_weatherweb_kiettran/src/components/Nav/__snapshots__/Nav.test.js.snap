// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Nav correctly 1`] = `
<div
  className="nav-container"
>
  <div
    className="nav-burger"
  >
    <span />
    <span />
    <span />
  </div>
  <div
    className="nav-location"
  >
    <div
      className="nav-location-name"
    >
      myENV
    </div>
    <div
      className="nav-location-dropdown"
    >
      <span>
        Current Location
      </span>
      <span
        className="icon-arrow"
      />
    </div>
  </div>
  <div
    className="nav-notification"
  />
</div>
`;
